docs/source/guide/index.rst

```diff
@@ -222,6 +222,7 @@ Step 2: Load the Online Key
     Choose 1/1 ONLINE key type [ed25519/ecdsa/rsa] (ed25519):
     Enter 1/1 the ONLINE`s private key path: tests/files/online.key
     Enter 1/1 the ONLINE`s private key password:
+    You can give a name to the key to have a reference in future:
```

Comment:
We could use something shorter and more straightforward for this sentence.
```suggestion
    [Optional] Give a name/tag to the key:
```

I'm suggesting “tag” because it is something commonly used nowadays.